Src/FluentAssertions/Collections/StringCollectionAssertions.cs

```diff
@@ -261,42 +261,6 @@ public AndConstraint<TAssertions> Contain(IEnumerable<string> expected)
             return base.Contain(expected);
         }
 
-        /// <summary>
-        /// Expects the current collection to contain the specified elements in any order. Elements are compared
-        /// using their <see cref="object.Equals(object)" /> implementation.
-        /// </summary>
-        /// <param name="expected">An <see cref="IEnumerable{T}" /> with the expected elements.</param>
-        /// <param name="because">
-        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
-        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
-        /// </param>
-        /// <param name="becauseArgs">
-        /// Zero or more objects to format using the placeholders in <paramref name="because" />.
-        /// </param>
-        public AndConstraint<TAssertions> Contain(IEnumerable<string> expected, string because = null,
```

Comment:
These two are removed as they now come for free through `Contain(IEnumerable<T>)` from the base class.